briefcase/django.py

```diff
@@ -78,13 +82,45 @@ def install_extras(self):
         # Install additional elements required for Django
         print(" * Installing extras...")
         print("   - Installing NPM requirements...")
-
-        npm = shutil.which("npm")
-        subprocess.Popen([npm, "install"], cwd=os.path.abspath(self.dir)).wait()
-
+        node_attributes = {
```

Comment:
well, I use it to get the download links, but I can delete the `'command_prefix'` key from the dictionary